Rust/Rust.sublime-syntax

```diff
@@ -633,24 +612,67 @@ contexts:
           pop: true
         - match: ','
           scope: punctuation.separator.rust
-        - include: type-any-identifier
+        - include: types-any
     - match: '\+'
       scope: keyword.operator.rust
+    - match: '\b_\b'
```

Comment:
What does this operator do? If it's the same as in other languages, I believe we scope it differently: https://github.com/sublimehq/Packages/commit/83a8f3e886b56c5c87957adb5c5f2ab61833f3b0

I realize you only moved this part, but would be nice to improve this at the same time if relevant :)

Reply:
It's used as a placeholder for type inference and thus has a special role in type signatures. I tried to formulate that into a proper `storage` scope, PTAL.

See also https://stackoverflow.com/questions/61950463/what-is-an-underscore-in-rust-type.